pyproject.toml

```diff
@@ -5,11 +5,8 @@ authors = [
     { name = "Raphael Nestler", email = "raphael.nestler@gmail.com" },
 ]
 
-requires-python = ">=3.8"
-dependencies = [
-    "cadquery == 2.3.1",
-    "numpy < 2",  # Dependency of cadquery, but not working with v2.x
-]
+requires-python = ">=3.9"
+dependencies = ["cadquery ~= 2.5"]
```

Comment:
Please point to a very specific release to ensure reproducibility of the STEP files.

Any is the numpy workaround no longer needed? Is it not installed anymore, or are the dependencies now pinning it to a compatible version?